cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -190,14 +118,14 @@ function wait_allocatable_resource {
 
 function deploy_multus {
   echo 'Deploying Multus'
-  _kubectl create -f $MANIFESTS_DIR/multus.yaml
+  _kubectl create -f $MANIFESTS_DIR/multus.yaml || return 1
```

Comment:
Isn't this script running with -e?

Reply:
It appears that  adding `|| exit 1`to helper function calls interfere with `-e` flag and somehow the error that occurred on the helper wont propagate.
I will remove `|| exit 1` when we call to helpers and remove`|| return 1` inside helpers since we dont need it.

Reply:
This is what i also shown an example of that happens,
please dont remove the `return 1`
as it will be a regression, you can either leave them or understand the root cause imo,
since function indeed should obey the `set -x`